libs/async_mpi/include/hpx/async_mpi/mpi_future.hpp

```diff
@@ -14,6 +14,7 @@
 #include <hpx/modules/threading_base.hpp>
 #include <hpx/mpi_base/mpi.hpp>
 #include <hpx/runtime_local/thread_pool_helpers.hpp>
+#include <mpi.h>
```

Comment:
```suggestion
#include <mpi_base/mpi.hpp>
```
`mpi.hpp` includes `mpi.h` but disables warnings that come otherwise from `mpi.h`.